HeliosEmbedded/Makefile

```diff
@@ -141,18 +140,32 @@ CFLAGS+=$(INCLUDES)
 ### SOURCE FILES ######
 #######################
 
+# Build directory for object files - default to ./build if not specified
+BUILD_DIR ?= ./build
+
+# Create architecture-specific build directories
+$(shell mkdir -p $(BUILD_DIR)/Helios)

```

Comment:
This is weird, so if you just run make on any target it will generate this folder...? Make clean generates the build dir?